src/autoscaler/eventgenerator/generator/evaluator_test.go

```diff
@@ -613,6 +607,7 @@ var _ = Describe("Evaluator", func() {
 							errorStr = errorStr + tmp
 						}
 						scalingEngine.RouteToHandler("POST", urlPath, ghttp.RespondWithJSONEncoded(http.StatusBadRequest, errorStr))
+						Expect(triggerChan).To(BeSent(triggerArrayGT))
```

Comment:
@julz @kanekoh  "PContext("when the scaling engine's response is too long", func()" is the pending case as it need to build a too long string to make the request failed and sometimes it panics. So it is marked as pended. Do you mean this case?